versions/factory/errors.go

```diff
@@ -0,0 +1,9 @@
+package factory
+
+import "errors"
+
+// ErrFileNotFound signals that the provided path is invalid and does not belong to a file or a directory
+var ErrFileNotFound = errors.New("file not found")
+
+// ErrFileIsNotADirectory signals that the file is not a directory
+var ErrFileIsNotADirectory = errors.New("file is not a directory")
```

Comment:
Maybe `path is not a directory`? I think it doesn't sound that weird :)

Reply:
renamed to `no directory at the given path` and also the variable